mock/py/mockbuild/buildroot.py

```diff
@@ -140,7 +160,7 @@ def initialize(self, prebuild=False, do_log=True):
             self._lock_buildroot(exclusive=True)
             self._init(prebuild=prebuild, do_log=do_log)
         except BuildRootLocked:
-            pass
+            self._init_locked()
```

Comment:
This was a missing bit for making #627 to work (22c8fdcbd0f1de50b942a5792a0fa6e88c620946 is the first needed patch).